tests/manifests/xsd/test_xsd.py

```diff
@@ -1089,3 +1081,79 @@ def test_xsd_enumeration(rc: RawConfig, tmp_path: Path):
         xsd_file.write(xsd)
     manifest = load_manifest(rc, path)
     assert manifest == table
+
+
+def test_duplicate_removal(rc: RawConfig, tmp_path: Path):
+    xsd = """
+    <xs:schema xmlns="http://eTaarPlat.ServiceContracts/2007/08/Messages" elementFormDefault="qualified" targetNamespace="http://eTaarPlat.ServiceContracts/2007/08/Messages" xmlns:xs="http://www.w3.org/2001/XMLSchema">
+        <xs:element name="getDocumentsByWagonResponse" nillable="true" type="getDocumentsByWagonResponse" />
+        <xs:complexType name="getDocumentsByWagonResponse">
+            <xs:sequence>
+                <xs:element minOccurs="0" maxOccurs="1" name="searchParameters" type="getDocumentsByWagonSearchParams" />
+                <xs:choice minOccurs="1" maxOccurs="1">
+                    <xs:element minOccurs="0" maxOccurs="1" name="klaida" type="Klaida" />
+                    <xs:element minOccurs="0" maxOccurs="1" name="extract" type="Extract" />
+                </xs:choice>
+            </xs:sequence>
+        </xs:complexType>
+        <xs:complexType name="getDocumentsByWagonSearchParams">
+            <xs:sequence>
+                <xs:element minOccurs="0" maxOccurs="1" name="searchType" type="xs:string" />
+                <xs:element minOccurs="0" maxOccurs="1" name="code" type="xs:string" />
+                <xs:element minOccurs="0" maxOccurs="1" name="RegDateFrom" type="xs:string" />
+                <xs:element minOccurs="0" maxOccurs="1" name="RegDateTo" type="xs:string" />
```

Comment:
In this test case, we need at least one element, because we are testing duplicate model removal, so there elements are not needed in this test.